app/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDao.java

```diff
@@ -53,9 +53,11 @@
         private static final String PUBLISHED_PROPERTY = "published";
```

Comment:
We should verify, before returning a survey, that it is associated to the study of the hostname. The code does not/cannot enforce that for a key-based retrieval.